### DISCUSSION TITLE
Job got scheduled but not triggered on schedule after redis reset

### DISCUSSION THREAD

**Discussion by aditisri07** [apache/airflow#23962] [Q&A]
Redis got reset.
Task was scheduled on airflow got queued but not triggered on schedule.
Once we force restarted worker the tasks that were queued started running. Trigger had no error in its logs.
Why did the not trigger the job even though it got connected to redis again after the reset in redis happened?

The docker service logs for airflow worker are attached.
[airflow_worker.txt](https://github.com/apache/airflow/files/8785932/airflow_worker.txt)

**Comment by potiuk** [comment]
Can you please describe your scenario in detail. I do not understand what you have done. Please be as precise as possible. If this is a reproducible case that you have good scenario for - please open an issue for it.
